wagtaildraftail/client/registry.js

```diff
@@ -0,0 +1,41 @@
+const _registry = {
+  decorators: {},
+  sources: {},
+  strategies: {},
+};
+
+const registerDecorator = (name, decorator) => {
+  _registry.decorators[name] = decorator;
+  console.log('decorator: ' + name);
+};
+
+const getDecorator = (name) => {
+  return _registry.decorators[name];
+};
+
+const registerSource = (name, source) => {
+  _registry.sources[name] = source;
+  console.log('source: ' + name);
+};
+
+const getSource = (name) => {
+  return _registry.sources[name];
+};
+
+const registerStrategy = (name, strategy) => {
+  _registry.strategies[name] = strategy;
+  console.log('strategy: ' + name);
+};
+
+const getStrategy = (name) => {
+  return _registry.strategies[name];
+};
+
+export {
+  registerDecorator,
+  getDecorator,
+  registerSource,
+  getSource,
+  registerStrategy,
+  getStrategy,
+}
```

Comment:
This really looks like one object rather than a bunch of named exports – could we `export default` a single object instead? Then in the respective import, `Object.assign` its methods to the `wagtailDraftail` export.